README.md

```diff
@@ -145,6 +144,14 @@ const perfume = new Perfume({
         myAnalyticsTool.track('elementTimingPageTitle', { duration: data });
         break;
       default:
+        // Step tracking
+        if( attribution?.category === 'step' ){
```

Comment:
@Cdub-9 should `metricName` be `markStep`?
I mean, after all it's a type of metric, and will make this experience much cleaner.

Reply:
Good callout yes, i will update the metric name to be `userJourneyStep` then have the `step_name` be in the `attribution` object .